raiden/transfer/mediated_transfer/mediator.py

```diff
@@ -980,7 +985,7 @@ def secret_learned(
 
 def mediate_transfer(
     state: MediatorTransferState,
-    possible_routes: List[RouteState],
+    candidate_route_states: List[RouteState],
```

Comment:
Please, persist this value, it then can be used on a refund (if there are more then one route in there)